README.md

```diff
@@ -1,11 +1,17 @@
 [![Build Status](https://travis-ci.org/OCA/margin-analysis.svg?branch=8.0)](https://travis-ci.org/OCA/margin-analysis)
```

Comment:
Take the occasion to add the runbot badge: https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md